weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>I'm not so sure about the terminology about "building" an agent here, like:</p><p> </p><blockquote>In this recipe, we will build a...</blockquote><p>Wdyt about "In this recipe, we will use the Weaviate <code>TransformationAgent</code> to enhance our data in Weaviate" or similar?</p><p> </p><p>(The agent here is pre-built, unlike building agents with frameworks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>